File: src/main/java/com/apiVKmanual/functions/other/Other.java
package com.apiVKmanual.functions.other;

public class Other {

    //-----------------рандом ID для вывода бота или прочего-----------------------//
    public int randomId(int sizeTable) {
        int a = 0; // Начальное значение диапазона - "от"
        int b = sizeTable-1; // Конечное значение диапазона - "до"
        return (a + (int)Math.round( (Math.random() * b)));
    }
}

File: src/main/java/com/sasfmlzr/apivk/functions/other/Other.kt
package com.sasfmlzr.apivk.functions.other

import kotlin.math.roundToInt

class Other {

    //-----------------рандом ID для вывода бота или прочего-----------------------//
    fun randomId(sizeTable: Int): Int {
        val a = 0 // Начальное значение диапазона - "от"
        val b = sizeTable - 1 // Конечное значение диапазона - "до"
        return a + (Math.random() * b).roundToInt()
    }
}
